spec/10-overview.md

```diff
@@ -32,7 +32,7 @@ Trace context is split into two individual propagation fields supporting interop
 
 Tracing tools can provide two levels of compliant behavior interacting with trace context:
 
-- At a minimum they MUST propagate the `traceparent` and `tracestate` headers and guarantee traces are not broken. This behavior is also referred to as forwarding a trace.
-- In addition they CAN also choose to participate in a trace by modifying the `traceparent` header and relevant parts of the `tracestate` header containing their proprietary information. This is also referred to as participating in a trace.
+- At a minimum they MUST propagate the `traceparent` and `tracestate` headers from incoming requests to all outgoing requests, guaranteeing that traces are not broken. This behavior is also referred to as forwarding a trace.
+- In addition they CAN also choose to participate in a trace by modifying the `traceparent` header and relevant parts of the `tracestate` header containing their proprietary information. This is also referred to as participating in a trace. Participating tracing tools SHOULD also add the `traceparent` header to all outgoing requests, even when the associated incoming request did not have it (this is referred to as starting a trace).
```

Comment:
If we are worried about changing semantics, then this addition here, that participating tools SHOULD start a trace when the incoming request did not have W3C trace context headers, might need to be discussed a bit. From my point of view it is in line with the intention of the spec, but the spec never spelled that out explicitly before (or I might have missed it). 

Do we want to mandate always start a trace? Is SHOULD appropriate here (vs. CAN or MUST)? 

WDYT?

Reply:
this is when tool decide to participate. Tool may decide not to participate in trace and do nothing. So no change for the spec from my POV

Reply:
Sure, this is only for participating tools. My point was that even for participating tools we didn't mandate to always start a trace until now. I think that was the mental model most folks had about this aspect, but it wasn't spelled out explicitly, I think.

But if you (and others) agree that this does not change semantics, I'm absolutely on board with that view. 👍

Reply:
The section you are changing starts with "interacting with trace context", which seems to imply there is an inbound context. If you're looking for completeness, there should be a separate section onwhat to do when there is no incoming trace context. In that situation both starting a trace and doing nothing should be valid options, IMO, i.e. the spec should not dictate starting a new trace.

Reply:
> The section you are changing starts with "interacting with trace context", which seems to imply there is an inbound context. 

Hm, I don't read it that way as starting a new trace is also a form of interacting with the trace context. But anyway, having this in a separate section might be better.

> IMO, i.e. the spec should not dictate starting a new trace.

There seems to disagreement about that? I understood @SergeyKanzhelev comments as being in support of that notion, that to _participate_ also means to _start_ a trace if there is non.

Given that we have the phrase "A tracing tool can choose to change this behavior for each individual request to a component it is monitoring." there is alway the option to act as a non-participating tool for that request.  

It is probably best to remove this change from this PR entirely. I might try to come up with a separate section about how to behave when there is no incoming trace context. I think agreeing on some explicit rules about that situation and adding them to the spec would be good, but it's out of scope for this change.

Reply:
I removed this change.